felix/fv/vxlan_test.go

```diff
@@ -85,12 +86,22 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ VXLAN topology before addin
 
 				topologyOptions = createBaseTopologyOptions(vxlanMode, enableIPv6, routeSource, brokenXSum)
 				topologyOptions.FelixLogSeverity = "Debug"
+
+				// Allow sub-contexts to modify the topology options before deploying the topology.
+				if modifyTopologyOpts != nil {
```

Comment:
I don't think this works, the BeforeEach in the nested block will run after this one so you'll see nil here.